agent/src/iconfucius/cli/balance.py

```
@@ -707,17 +737,24 @@ def _format_holdings_table(all_data: list, btc_usd_rate: float | None,
     headers = ["Bot", "ckBTC"] + token_headers
     rows = []
     total_odin_sats = 0
+    any_odin_unknown = False
     total_token_balances = {ticker: 0 for ticker in all_tickers}
     total_token_divisibility = {ticker: 0 for ticker in all_tickers}
     total_token_value_sats = {ticker: 0.0 for ticker in all_tickers}
 
     for d in all_data:
-        total_odin_sats += int(d.odin_sats)
+        if d.odin_sats is None:
+            any_odin_unknown = True
```

Ruff (0.15.2)

[error] Local variable `any_odin_unknown` is assigned to but never used

Remove assignment to unused variable `any_odin_unknown`

(F841)